schemas/event.yml

```diff
@@ -27,6 +27,14 @@
         This is normally defined by the user.
       example: nginx-stats-metrics
 
+    - name: action
+      type: keyword
+      description: >
+        The action taken by the event.
+
+        This is normally defined by the user.
```

Comment:
Could you add a use case example here like one of the example discussed in https://github.com/elastic/ecs/issues/21.